src/app/technical-record/adr/adr-details/adr-details.component.html

```diff
@@ -13,7 +13,7 @@ <h2 class="govuk-heading-m">ADR details</h2>
     <div class="govuk-summary-list__row">
       <dt class="govuk-summary-list__key">Date processed</dt>
       <dd id="test-ADR-approvalDate" class="govuk-summary-list__value">
-        {{ adrDetails.vehicleDetails?.approvalDate | DefaultNullOrEmpty }}
+        {{ adrDetails.vehicleDetails?.approvalDate ? (adrDetails.vehicleDetails?.approvalDate | date: 'dd/MM/yyyy') : '-' }}
```

Comment:
Can we use a pipe instead for this please? If easy to do, I believe we could chain Pipes.
It would be best avoiding inline Js in the markup

Reply:
Hi @JonLazariniDD  i think this if fine here. The nullable operator `?.` Serves as a shortcut in this case. Having another will be over kill